downstream/modules/automation-hub/proc-add-group-to-container-repo.adoc

```diff
@@ -26,7 +26,7 @@ Provide access to your container repository to users who need to work the images
 
 . Navigate to *Execution Environments*.
 . Select your container repository.
-. Click *Edit* at the top right of your window.
+. Click btn:[Edit] at the top right of your window.
```

Comment:
This violates accessibility by using directional instructions with "top right of your window". I think we need to see the UI to determine how this could be rephrased to eliminate directional information.

Reply:
Leaving this comment for now while we await review of the UI.